### PR TITLE
Smoke test are now required for apps

### PR DESCRIPTION
**Pull request by wheresrhys**
@matthew-andrews @TheoLeanse 

Also @aintgoin2goa Can you think of anywhere else I'd need to add this for ship/flat etc. They all look like they only ever use deploy internally, but not suer if I missed something

### CODE REVIEW

File: tasks/deploy.js [hunk 1]
```diff
@@ -55,11 +65,7 @@ module.exports = function (program, utils) {
 		.command('deploy [app]')
 		.description('runs haikro deployment scripts with sensible defaults for Next projects')
 		.option('-s, --skip-gtg', 'skip the good-to-go HTTP check')
-		.option('--gtg-urls <urls>', 'Comma separated list of urls to check before concluding the app is ok (these are in addition to __gtg)', utils.list)
```
Comment (matthew-andrews) [inline tasks/deploy.js]: could we have a `--skip-smoke` option for cases so we can opt out of this feature?

Comment (wheresrhys) [inline tasks/deploy.js]: why? which cases?

Comment (matthew-andrews) [inline tasks/deploy.js]: apps that don't have web processes, e.g. flag notifier

Comment (wheresrhys) [inline tasks/deploy.js]: in that case --skip-gtg covers it

Comment (wheresrhys) [inline tasks/deploy.js]: which I could alias to --skip-smoke-tests

Comment (matthew-andrews) [inline tasks/deploy.js]: ah ok yep

Comment (matthew-andrews) [inline tasks/deploy.js]: well

Comment (matthew-andrews) [inline tasks/deploy.js]: yes i guess so

**Pull request merged**